CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+### citus v6.0.1 (November 29, 2016) ###
+
+* Fixes a bug preventing DML queries while creating colocated table
+
+* Fixes a bug causing incorrectly passing down NULL parameters as text
```

Comment:
Not syntactic. Maybe _Fixes a bug that caused NULL parameters to be incorrectly passed as text_?